dspace-api/src/test/data/dspaceFolder/config/local.cfg

```diff
@@ -108,3 +108,10 @@ plugin.sequence.java.util.Collection = \
     java.util.LinkedList, \
     java.util.Stack, \
     java.util.TreeSet
+
+
+# Enable IRUS tracker for IRUS tests
+stats.tracker.enabled = true
```

Comment:
This doesn't need to be added. It can be initialized in the tests using `configurationService.setProperty()` rather than adding it here.  Adding it here has global effect (across all tests) when we only need it for a few

Reply:
Moved the enabling to the tests.